website/_includes/base-resp.html

```diff
@@ -19,6 +19,11 @@
 {% block site_header %}
   <header id="masthead">
 
+  {% block site_header_logo %}
+  <h2><a href="{{ url('thunderbird.index') }}">{{ high_res_img('thunderbird/template/logo-wordmark.png', {'alt':
+      'Thunderbird', 'width': '281', 'height': '66'}) }}</a></h2>
```

Comment:
No forced line-length wrapping in HTML templates, please.

Reply:
line-height? I didn't add any line-height here, I simply moved this block before the `nav` block.
What do you mean?

Reply:
I'm talking about the code line length, not line-height property. :) Don't wrap to 120 characters or whatever that is, just leave the `<a href>` on one line like it originally was.

Reply:
Oh, you mean wrapping code at a character limit. Sorry about that, my code editor did it automatically when I moved that block up.
It's fixed now.